data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -978,7 +978,7 @@
 
                 [message]
                     speaker=Elenia
-                    message= _ "Oh, just a few tricks I know, otherwise I probably would be dead by now. But let’s get back at that disgusting skeleton."
+                    message= _ "Oh, just a few tricks I know, otherwise I probably would be dead by now. Now where's that that disgusting skeleton?"
```

Comment:
"that that" and non-typographic apostrophe

Reply:
Again!